test/end-to-end/pages/executors/number.js

```diff
@@ -1,17 +1,23 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/executors/number');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const pagePath = require('app/steps/ui/executors/number');
+const numberContentEn = require('app/resources/en/translation/executors/number');
+const numberContentCy = require('app/resources/cy/translation/executors/number');
 
-module.exports = async function (totalExecutors) {
+module.exports = async function (language = 'en', totalExecutors) {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const numberContent = language === 'en' ? numberContentEn : numberContentCy;
 
-    await I.checkPageUrl('app/steps/ui/executors/number');
-    await I.waitForText(content.checklist1Header, config.TestWaitForTextToAppear);
+    await I.seeInCurrentUrl(pagePath.getUrl());
+    if (language === 'en') {
+        await I.waitForText(numberContent.checklist1Header, config.TestWaitForTextToAppear);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages